protocol/swap/swap_manager.go

```diff
@@ -188,8 +188,8 @@ func (m *manager) GetOngoingSwapOfferIDs() ([]*types.Hash, error) {
 	defer m.RUnlock()
 
 	offerIDs := make([]*types.Hash, 0, len(m.ongoing))
-	for _, s := range m.ongoing {
-		offerIDs = append(offerIDs, &s.OfferID)
+	for i := range m.ongoing {
+		offerIDs = append(offerIDs, &m.ongoing[i].OfferID)
 	}
```

Comment:
This change was only to make the linter happy. The original code looks safe to me.